downstream/modules/platform/proc-central-auth-install-keycloak.adoc

```diff
@@ -0,0 +1,191 @@
+[id="proc-central-auth-install-keycloak"]
+
+= Install Keycloak with Ansible
+
+When you have installed the collection and dependencies necessary for your network to run the installation of Keycloak performs the following actions on  the system running it:
+
+* Creates user and group accounts
+
+* Downloads the installation archive from the Keycloak website
+
+* Unarchives the content you downloaded while ensuring that the files are associated with the correct users and privileges
+
+* Verifies that your version of Java Virtual Machine is up-to-date
+
+* Integrates Keycloak into the host service management system (e.g., systemd daemon)
+
+[NOTE]
+====
+Both user and group account names are keycloak by default.
+====
+
+. Create the following playbook:
++
+- name: Playbook for Keycloak Hosts
+  hosts: all
+  vars:
+    keycloak_admin_password: "remembertochangeme"
+    keycloak_realm: TestRealm
+  collections:
+    - middleware_automation.keycloak
+  roles:
+    - keycloak
+
+
+The configuration adds {PlatformNameShort}'s collection for Keycloak to the list the playbook uses, and the associated `middleware_automation.keycloak.keycloak`.
+
+[NOTE]
+====
+The password at the end of the playbook creation at the variable ˋkeycloak_admin_passwordˋ must be a minimum of 12 characters.
+====
+
+[IMPORTANT]
+====
```

Comment:
Are these examples meant to appear inside a table? It looks like that's how it's rendering.